kit/src/lib/Docstring.svelte

```diff
@@ -136,7 +149,7 @@
 			</p>
 			<ul class="px-2">
 				{#each parametersDescription as { anchor, description }}
-					<li class="text-base !pl-4 my-3">
+					<li class="text-base !pl-4 my-3 rounded {hashlink === anchor && bgHighlightClass}">
```

Comment:
`{hashlink === anchor && bgHighlightClass}` will probably add the `"false"` css class when it's not an hashlink.

```suggestion
					<li class="text-base !pl-4 my-3 rounded {hashlink === anchor ? bgHighlightClass : ''}">
```

same below

Reply:
handled in https://github.com/huggingface/doc-builder/pull/177/commits/69e2011ce57ada67f16b7a02fa8034ce113553d5